plugins/events-backend/README.md

```diff
@@ -44,6 +44,18 @@ Then update plugin environment to include the event broker.
 +  eventBroker: EventBroker;
 ```
 
+To enable events also to frontend, you must pass signal service to the broker:
+
+```diff
+// packages/backend/src/index.ts
++  const eventBroker = new DefaultEventBroker(
++    root.child({ type: 'plugin' }),
++    signalsClientManager.forPlugin('events').getClient(),
```

Comment:
This is something that should be discussed. Is this application developers responsibility, should this be allowed at all, or should the events backend somehow limit which events are published also to the frontend?